apps/console/src/public/deployment.config.json

```diff
@@ -736,6 +736,22 @@
                     ]
                 }
             },
+            "dynamicClientRegistration": {
+                "disabledFeatures": [],
+                "enabled": true,
+                "featureFlags": [],
+                "scopes": {
+                    "feature": [
+                        "console:applications"
+                    ],
+                    "read": [
+                        "internal_config_view"
+                    ],
+                    "update": [
```

Comment:
Lets add all the possible operations and keep empty

Reply:
Updated via c37b6504e865b48e618022a346ff39df46ae2584